src/aurelia-binding.d.ts

```diff
@@ -509,6 +512,28 @@ export declare interface InternalCollectionObserver {
    * @param callable A callable object.
    */
   unsubscribe(context: any, callable: Callable): void;
+  /**
+   * Adds a change record to the collection observer.
+   * @param changeRecord 
+   */
+  addChangeRecord(changeRecord: ICollectionObserverSplice): void;
+  /**
+   * This will flush the change records of this observer and call any subscribers if applicable.
+   */
+  flushChangeRecords(): void;
+  /**
+   * Reset the observer to the passed collection and call any subscribers with changes between the current collection and the reset collection.
+   * @param oldCollection 
+   */
+  reset(oldCollection): void;
+  /**
+   * Get a length observer for this collection.
+   */
+  getLengthObserver(): any;
+  /**
+   * This will call subscribers notifying of changed records.
+   */
+  call(): void;
```

Comment:
I don't think this should be exposed as it's a special interface for the task queue to be called. Especially when there is already the other mechanism to flush. Would just confuse users, I think.

Reply:
I agree, I will revert the addition of this.